src/main/java/project/controller/DriverRiderController.java

```diff
@@ -30,7 +31,7 @@ public DriverRiderController(UserService userModel, MockDataService mockDataServ
     }
 
     @RequestMapping(value = "/driverrider", method = RequestMethod.GET)
```

Comment:
@pajdak3 now the `/driverrider` returns an User object with phone number, if we have a phone number in the database for that user, so you can use that to pre-fill the phone number when registering rider / driver